client/src/pages/NewsPage/NewsPage.jsx

```diff
@@ -1,11 +1,75 @@
+	},
+	{
+		title: "Surgical Procedures",
+		image:
+			"https://i0.wp.com/post.healthline.com/wp-content/uploads/2020/09/Female_Doctor_Daughter_Mother_1296x728-header-1296x729.jpg?w=1155&h=2268",
+		description:
+			"Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+	},
+	{
+		title: "Diagnostic Imaging",
+		image:
+			"https://s3-prod.modernhealthcare.com/s3fs-public/SPONSORED_170619878_AR_-1_RXMUPRMWBUGI.jpg",
+		description:
+			"Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+	},
+];
 
 const NewsPage = () => {
+	const [searchValue, setSearchValue] = useState("");
+
+	const handleSearch = (event) => {
+		setSearchValue(event.target.value);
+	};
+
+	const filteredNews = newsData.filter((item) => {
+		const titleMatches = item.title
+			.toLowerCase()
+			.includes(searchValue.toLowerCase());
+		const contentMatches = item.description
+			.toLowerCase()
+			.includes(searchValue.toLowerCase());
+		return titleMatches || contentMatches;
+	});
 	return (
 		<>
-			<h1>News Page</h1>
+			<h1>News</h1>
+			<Line />
+			<div className="tools-bar">
+				<input
+					type="text"
+					placeholder="Search"
+					value={searchValue}
+					onChange={handleSearch}
+				/>
+				<p>{filteredNews.length} news found</p>
+			</div>
```

Comment:
for accessibility let's make this a form

```suggestion
			<form className="search-form">
				<input
					type="text"
					placeholder="Search"
					value={searchValue}
					onChange={handleSearch}
				/>
				<p>{filteredNews.length} news found</p>
			</form>
```